main.js

```diff
@@ -0,0 +1,8 @@
+//QUERY SELECTORS
+const totalCost = document.getElementById('totalCost');
+const recipientName = document.getElementById('recipientName');
+const wantedItem = document.getElementById('wantedItem');
+const individualCost = document.getElementById('indivCost');
+const checkbox = document.getElementById('checkBox');
+
+//FUNCTIONS
```

Comment:
I am going to try doing the fetch call in the dev tools console first, so I can see what comes out.